lite/kernels/npu/graph_compute.cc

```diff
@@ -30,10 +30,14 @@ void GraphCompute::PrepareForRun() {
   auto& ctx = this->ctx_->template As<NPUContext>();
   auto& param = this->Param<param_t>();
 
-  exec_ = ctx.client(param.model_name);
-  CHECK(exec_);
+  CHECK(param.weight);
+  CHECK(lite::npu::LoadModel(*param.weight, &model_client_, &model_name_));
+  // TODO(hong19860320): find an good way to free the model data.
+  param.weight->Resize({1});
```

Comment:
why 1？

Reply:
No interface exists to free the data of tensor, so I resize the dim to 1 and change target to force it to realloc a small size memory.